### PR TITLE
feat: improve compatibility with `act`, integration tests

### PR DESCRIPTION
**Pull request by philipreinken**
Depends on:
- [x] https://github.com/shopware/github-actions/pull/39

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -193,16 +193,32 @@ runs:
...
     - name: Install Shopware
       if: inputs.install == 'true'
       shell: bash
       working-directory: ${{ inputs.path }}
-      run: bin/console system:install --create-database --drop-database --basic-setup --force --shop-locale=${{ inputs.install-locale }} --shop-currency=${{ inputs.install-currency }}
+      run: |
+        composer run init:db -- --shop-locale=${{ inputs.install-locale }} --shop-currency=${{ inputs.install-currency }}
+        composer run build:js
+        bin/console theme:change --sync --all Storefront
+        bin/console assets:install --env="${{ inputs.env }}"
```
Comment (shyim) [inline action.yml:218]: I would not build the admin/storefront by default

Comment (philipreinken) [reply action.yml:218]: Right, this shouldn't occur with every execution; I'll change it!

File: action.yml [hunk 2]
```diff
@@ -193,16 +193,44 @@ runs:
...
+      run: |
         echo "DATABASE_URL=mysql://root@127.0.0.1/shopware" >> "$GITHUB_ENV"
+      if: ${{ env.DATABASE_URL == '' }} # Don't override if already set
+
+    - name: Configure HMAC JWT
+      shell: bash
+      working-directory: ${{ inputs.path }}
+      run: |
+        yq -n '.shopware.api.jwt_key.use_app_secret = true' | tee config/packages/zz-shopware.yaml
+      # if: ${{ steps.version.outputs.shopware-major-version > 5 }} # TODO: Add support for this in the shopware-version action
```
Comment (philipreinken) [inline action.yml:209]: https://github.com/shopware/github-actions/pull/39

Comment (philipreinken) [reply action.yml]: Nevermind, this won't work with dev-branches.

**Review by mkraeml — APPROVED** [review]

**Pull request merged**